libcextract/SymbolExternalizer.cpp

```diff
@@ -37,10 +37,12 @@
    the identifier.  */
 #define TOKEN_VECTOR " ().,;+-*/^|&{}[]<>^&|\r\n\t"
 
-static std::vector<SourceRange>
-Get_Range_Of_Identifier_In_SrcRange(const SourceRange &range, const char *id)
+/* Return the ranges for all identifiers on the ids vector */
+template <typename T>
+static std::vector<std::pair<std::string, SourceRange>>
+Get_Range_Of_Identifier(const SourceRange &range, T ids)
```

Comment:
It should be `T &ids`, or maybe `const T &ids`. The way it is written you will copy the datastructure, which is not needed.

Reply:
done